modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/events/LogoutPreAction.java

```diff
@@ -38,13 +38,15 @@ public void run(
 			for (Cookie cookie : cookies) {
 				String name = cookie.getName();
 
-				if (name.startsWith(
-						CommerceOrder.class.getName() + StringPool.POUND)) {
-
+				if (name.startsWith("COMMERCE_COMPARE")) {
```

Comment:
**[Non-blocking]** Since the consequent is the same, could this be simplified with an `OR`?